.eslintrc.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
     ],
     'react-hooks/rules-of-hooks': 'error',
     'react-hooks/exhaustive-deps': 'warn',
+    'jsx-a11y/no-autofocus': 'off',
```

Comment:
Can we not do this? Using `autofocus` causes a poor user experience by bypassing information at the top of the screen, forcing users to navigate backward to understand the current context.